### PR TITLE
Added the delete_credentials method to the SubscriberApi class to allow deleting user credentials.

### PR DESCRIPTION
**Pull request by MuhammadNFadhil**

### CODE REVIEW

**Comment by MuhammadNFadhil** [comment]
@Cliftonz  @unicodeveloper

**Comment by Cliftonz** [comment]
@MuhammadNFadhil Looks like the CI is failing I would take a look at the output see what is going on and make a push.

File: novu/api/subscriber.py [hunk 1]
```diff
@@ -123,6 +123,22 @@ def credentials(
...
+        """Delete subscriber credentials such as slack and expo tokens.
+
+        Args:
+            subscriber_id: The subscriber identifier
+            provider_id: The provider name (e.g: slack)
+        """
+
+        return SubscriberDto.from_camel_case(
+            self.handle_request("DELETE", f"{self._subscriber_url}/{subscriber_id}/credentials/{provider_id}")
+        )
```
Comment (ryshu) [inline novu/api/subscriber.py:140]: It doesn't match API SPEC, which indicate that this action doesn't return any data

https://api.novu.co/api/#/Subscribers/SubscribersController_deleteSubscriberCredentials

Comment (ryshu) [reply novu/api/subscriber.py:140]:
```suggestion
        self.handle_request("DELETE", f"{self._subscriber_url}/{subscriber_id}/credentials/{provider_id}")
```

File: novu/api/subscriber.py [hunk 2]
```diff
@@ -123,6 +123,22 @@ def credentials(
             self.handle_request("PUT", f"{self._subscriber_url}/{subscriber_id}/credentials", payload).get("data", {})
         )
 
+    def delete_credentials(
+        self,
+        subscriber_id: str,
+        provider_id: str,
+    ) -> SubscriberDto:
```
Comment (ryshu) [inline novu/api/subscriber.py:130]:
```suggestion
    ) -> None:
```

**Review by ryshu — CHANGES_REQUESTED** [review]
Like @Cliftonz say, missing unit tests

Please take care to rename your commit following the contribution guidelines: https://github.com/novuhq/novu-python/blob/main/CONTRIBUTING.rst

**Comment by ryshu** [comment]
Hi @MuhammadNFadhil ,

Ok for the code but tests are still missions. You need to implement them.

**Comment by ryshu** [comment]
@MuhammadNFadhil @Cliftonz Updated to match all requirements (CI/CD, unit tests)

Ready to be "rebase and merge" in alpha

**Review by ryshu — APPROVED** [review]

**Pull request merged**